main.js

```diff
@@ -2,3 +2,28 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+const characterList = $('.item');
+
+function getCharacters() {
+    for (let i = 0; i <= 4; i++) {
+      axios({
+        method: 'get',
+        url: `https://rickandmortyapi.com/api/character/${i + 1}`
+      })
+        .then(res => {
+          const characterName = $(characterList[i].children[0]);
```

Comment:
here you can also use to create a new element which you can append into your HTML code
```
const container = qurrySelector('.container')
const newElement = document.createNewElement()
 container.append(newElement)
```